backend/app/views/spree/admin/shared/_refunds.html.erb

```diff
@@ -1,11 +1,11 @@
 <table class="index" id='payments' data-order-id='<%= @order.number %>'>
   <thead>
     <tr data-hook="refunds_header">
-      <th><%= "#{Spree.t('date')}/#{Spree.t('time')}" %></th>
+      <th><%= Spree::Refund.human_attribute_name(:created_at) %></th>
       <th><%= Spree.t(:payment_identifier) %></th>
```

Comment:
Should this be Spree::Payment.human_attributes_name(:identifier) ? or something fancier including the model?